templates/zerver/help/analytics.md

```diff
@@ -0,0 +1,83 @@
+# Analytics
+
+Zulip provides analytics for tracking various usage statistics about you and
+your organization.
+So, you can get a holistic view of your organization's communications.
+
+Analytics can be accessed via the [stats](/stats) page.
+
+### Messages Sent Over Time
+
+You can view number of messages have been sent over a particular time frame.
+
+* You can change the **Aggregation** to view the number of messages sent on an
+hourly, daily, weekly basis or the cumulative amount.
+
+    ![Aggregation](/static/images/help/stats-aggregation.png)
+
+* You can change the time frame of messages to **Last 2 months**, **Last 6 months**
+or **All time**.
+
+    ![Sliding window](/static/images/help/stats-sliding-window.png)
+
+    A custom time frame is also possible by clicking and sliding the handle in
+    the bottom window.
+
+* By default the number of messages sent by **Humans** (users) is shown, but
+you can also view the message count of bots by clicking on **Bots** at the
```

Comment:
either "the message sent count" or "the number of messages sent by bots" for consistency